### DISCUSSION TITLE
sum-numbers ödevi düzenleme hatası

### DISCUSSION THREAD

**Discussion by MohamedHassan1606** [19/classroom#43] [Anonim] [answered]
İyi akşamlar hocam

Ödevi ikinci kez küçücük ekleme yaptıktan sonra kaydettim ancak main göbeğinde iki kod yazım gözümektedir. Onu düzenlemeye çalıştım ancak düzenleme tuşu yoktur hocam. Bunu son kez düzeltmem için neyi yapmam gerektiğini düşünüyorsunuz hocam?

Teşekkürler

**Comment by MohamedHassan1606** [comment]
Hocam, Ödevin kurallarını dikkate alarak (puts main) write code here kısmında yanı main göbeğin içine aldım. Ancak preview changes tuşunu bastığımda görüyorum ki (end)tan sonra da puts main var. o da kodumu bölüyor ve kaydetsemde koddun doğru yoldan gidiyor. bu sorunu nasıl çözeyim.

**Comment by roktas** [comment]
@MohamedHassan1606 Daha önemli bir sorunun var: ödevi anlamamışsın.  2-3 sayının toplamı değil girilen tüm sayıların toplamı isteniyor.

**Reply by MohamedHassan1606** [reply]
Hocam düzeltim ama güncelledigim kodu hala bır sıkıntı var, lütfen bakar mısınız hocam?

**Comment by roktas** [comment, marked as answer]
@MohamedHassan1606 uygulamaya katıl ve hocalarına sor.  Burada yazılamayacak kadar hatan var.  Ve hala soruyu anlamamışsın.  Senden kafana göre 4 veya 5 sayının toplamını bulman istenmiyor; ne kadar sayı verilirse hepsini toplaman isteniyor.

Ayrıca lütfen şu hata mesajlarını hiç mi okumadın?  https://github.com/19-22s-ruby-2/sum-numbers-MohamedHassan1606/runs/5659716808
